e2e_tests/tests/deploy/test_local.py

```diff
@@ -165,13 +165,10 @@ def test_agent_config_path() -> None:
 
     # Validate CLI flags overwrite config file options.
     agent_name += "-2"
-    agent_up(
-        ["--agent-name", agent_name, "--agent-config-path", etc_path, "--agent-label", "cli-flag"]
-    )
+    agent_up(["--agent-name", agent_name, "--agent-config-path", etc_path])
```

Comment:
should this PR remove ```--agent-label``` from ```det deploy local``` or is it being left intentionally for backwards compatibility?

Reply:
ah, I missed it. will remove.